### DISCUSSION TITLE
Github actions powered wintuner

### DISCUSSION THREAD

**Discussion by 1ARdotNO** [svrooij/WinTuner#156] [Show and tell]
I built some automations around wintuner to be able to run it from Github actions, useful for those of us that does not have Azure to run it in, or simply want to run it for free either with selfhosted runners or using included minutes or free minute from Githib Actions :- ) 

I made this for my own use, but maybe someone else will find it useful also;

https://github.com/1ARdotNO/wintuner-actions

**Comment by svrooij** [comment]
You can make this even better by using federated credentials... That way you won't even need a secret that might expire

**Reply by 1ARdotNO** [reply]
That is a good idea, 
I'll look into that sometime in the new year🎇
